TLM/TLM/UI/SubTools/JunctionRestrictionsTool.cs

```diff
@@ -87,6 +87,10 @@ public JunctionRestrictionsTool(TrafficManagerTool mainTool)
             ushort updatedNodeId = 0;
             bool handleHovered = false;
             bool cursorInPanel = IsCursorInPanel();
+            PrioritySigns.PrioritySignsOverlay overlay = new PrioritySignsOverlay(
```

Comment:
Why did you rename `Overlay.cs` to `PrioirtySignsOverlay.cs` instead of moving it to `UI.Helpers` directory?
you call it `PrioirtySignsOverlay` but its only called by Junction Restriction tool (for now anyway).